sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/CosmosAsyncContainer.java

```diff
@@ -2549,6 +2552,36 @@ public Mono<List<FeedRange>> getFeedRanges() {
         return this.getFeedRanges(true);
     }
 
+    public Mono<List<FeedRange>> getOverlappingFeedRanges(FeedRange feedRange) {
```

Comment:
Didn't we say this should be internal/ImplementationBridgeHelper?

Reply:
I really don't think it should be public - but even if we decide to make it public it would need javaDoc and explaantion when customers would want to use it?

Reply:
Never mind - don't think you made any changes after the discussion yesterday :-)

Reply:
yea - now have pushed the change